scripts/upload_test_resources.py

```diff
@@ -0,0 +1,151 @@
+from quilt3 import Package
+
+from aicsimageio import __version__
+
+###############################################################################
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)4s: %(module)s:%(lineno)4s %(asctime)s] %(message)s",
+)
+log = logging.getLogger(__name__)
+
+###############################################################################
+# Args
+
+
+class Args(argparse.Namespace):
+    def __init__(self):
+        self.__parse()
+
+    def __parse(self):
+        # Setup parser
+        p = argparse.ArgumentParser(
+            prog="upload_test_resources",
+            description=(
+                "Upload files used for testing this project. This will upload "
+                "whatever files are currently found in the `tests/resources` directory."
+                "To add more test files, simply add them to the `tests/resources` "
+                "directory and rerun this script."
+            ),
+        )
+
+        # Arguments
+        p.add_argument(
+            "--dry-run",
+            action="store_true",
+            help=(
+                "Conduct dry run of the package generation. Will create a JSON "
+                "manifest file of that package instead of uploading."
+            ),
+        )
+        p.add_argument(
+            "--debug",
+            action="store_true",
+            help="Show traceback if the script were to fail.",
+        )
+
+        # Parse
+        p.parse_args(namespace=self)
+
+
+###############################################################################
+# Build package
+
+
+def upload_test_resources(args: Args):
+    # Try running the download pipeline
+    try:
+        # Get test resources dir
+        resources_dir = (
+            Path(__file__).parent.parent / "aicsimageio" / "tests" / "resources"
+        ).resolve(strict=True)
+
+        # Report with directory will be used for upload
+        log.info(f"Using contents of directory: {resources_dir}")
+
+        # Create quilt package
+        package = Package()
+        package.set_dir("resources", resources_dir)
+
+        # Report package contents
+        log.info(f"Package contents: {package}")
+
+        # Construct package name
+        package_name = "aicsimageio/test_resources"
+
+        # Check for dry run
+        if args.dry_run:
+            # Attempt to build the package
+            built = package.build(package_name)
+
+            # Get resolved save path
+            manifest_save_path = Path("upload_manifest.jsonl").resolve()
+            with open(manifest_save_path, "w") as manifest_write:
+                package.dump(manifest_write)
+
+            # Report where manifest was saved
+            log.info(f"Dry run generated manifest stored to: {manifest_save_path}")
+            log.info(f"Completed package dry run. Result hash: {built.top_hash}")
+
+        # Upload
+        else:
+            # Get upload confirmation
+            confirmation = None
```

Comment:
You might want a confirm argument (like `conda install <package> -y`) which will override this/remove the need for user input

Reply:
I like that!